modules/nvidia_plugin/src/cuda_compiled_model.hpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/runtime/icompiled_model.hpp"
+#include "openvino/runtime/threading/itask_executor.hpp"
+
+#include "cuda_async_infer_request.hpp"
+#include "cuda_config.hpp"
+#include "cuda_graph.hpp"
+#include "cuda_infer_request.hpp"
+#include "cuda_op_buffers_extractor.hpp"
+#include "memory_manager/cuda_device_mem_block.hpp"
+#include "memory_manager/cuda_memory_manager.hpp"
+#include "memory_manager/cuda_memory_pool.hpp"
+#include "memory_manager/model/cuda_memory_model.hpp"
+#include "ops/subgraph.hpp"
+
+class ExecNetworkTest;
```

Comment:
Why do we need this here?
Can it be renamed to `CompiledModelTest`?

Reply:
Removed